projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -546,3 +546,16 @@ export function canShowLanguagePicker(context: AcaRuleContext): boolean {
 export function canShowLogout(context: AcaRuleContext): boolean {
   return !context.withCredentials;
 }
+
+/**
+ * Checks if user is library man
```

Comment:
Library man ? :)

Reply:
Always good to know if a user is a good Library Man....ager!